src/docs/user/references/GenericInformation.adoc

```diff
@@ -432,18 +440,25 @@ The Docker Compose generic information should be defined at *task-level* but can
 
-* `docker-actions` : actions to perform. A comma separated list of possible actions (build, run, exec, stop, rmi). Default is `build,run,stop,rmi`.
-* `docker-image-tag` : tag identifying the docker image. Default is `image_${PA_JOB_ID}t${PA_TASK_ID}`
-* `docker-container-tag` : tag identifying the docker container. Default is `container_${PA_JOB_ID}t${PA_TASK_ID}`
-* `docker-build-options` : options given to the `docker build` command.
-* `docker-run-options` : options given to the `docker run` command.
-* `docker-exec-command` : command given to `docker exec`, if used in `docker-actions`. If the command contains spaces, `docker-file-options-split-regex` should be used to split command parameters.
-* `docker-exec-options` : options given to the `docker exec` command. Default is `-t` (which should always be included).
-* `docker-stop-options` : options given to the `docker stop` command.
-* `docker-rm-options` : options given to the `docker rm` command.
-* `docker-rmi-options` : options given to the `docker rmi` command.
-* `docker-file-options-split-regex` : declare a string to be used as options separator, instead of the `space` character.
+* `docker-actions`: actions to perform. A comma separated list of possible actions (build, run, exec, stop, rmi). Default is `build,run,stop,rmi`.
+* `docker-image-tag`: tag identifying the docker image. Default is `image_${PA_JOB_ID}t${PA_TASK_ID}`
+* `docker-container-tag`: tag identifying the docker container. Default is `container_${PA_JOB_ID}t${PA_TASK_ID}`
+* `docker-build-options`: options given to the `docker build` command.
+* `docker-run-options`: options given to the `docker run` command.
+* `docker-exec-command`: command given to `docker exec`, if used in `docker-actions`. If the command contains spaces, `docker-file-options-split-regex` should be used to split command parameters.
+* `docker-exec-options`: options given to the `docker exec` command. Default is `-t` (which should always be included).
+* `docker-stop-options`: options given to the `docker stop` command.
+* `docker-rm-options`: options given to the `docker rm` command.
+* `docker-rmi-options`: options given to the `docker rmi` command.
+* `docker-file-options-split-regex`: declare a string to be used as options separator, instead of the `space` character.
 
 See <<../user/ProActiveUserGuide.adoc#_dockerfile,Dockerfile script language>> for further information.
 
-The Dockerfile generic information should be defined at *task-level*. Some (docker-file-options-split-regex, or command options) may be defined at *job-level* to apply to all tasks of the workflow.
\ No newline at end of file
+The Dockerfile generic information should be defined at *task-level*. Some (docker-file-options-split-regex, or command options) may be defined at *job-level* to apply to all tasks of the workflow.
```

Comment:
the task level

Reply:
@mboussaa Should I change both *task-level* and  *job-level* ?

Reply:
yes I think so